packages/augur-core/source/contracts/libraries/Ownable.sol

```
@@ -40,8 +40,9 @@ contract Ownable is IOwnable {
         if (_newOwner != address(0)) {
             onTransferOwnership(owner, _newOwner);
             owner = _newOwner;
+            return true;
         }
-        return true;
+        return false;
```

It feels like this function should `require(_newOwner != address(0))` rather than returning false on failing to do the thing you told it to do.